contrib/pyln-proto/pyln/proto/zbase32.py

```diff
@@ -23,34 +23,69 @@
 ]
 
 
-def bitarray_to_u5(barr):
-    assert len(barr) % 5 == 0
+def _message_to_bitarray(message):
+    barr = bitstring.ConstBitStream(message)
+    filling_len = len(barr) % 5
+    if filling_len:
+        # The bitarray length has to be multiple of 5. If not, it is right-filled with zeros.
+        filling = '0'*(5-filling_len)
+        barr = bitstring.ConstBitStream(bin=f"{barr.bin}{filling}")
```

Comment:
Not sure we have f-string support in the lightning codebase yet. We only specified a minimum version of python in the `lightningd/plugins` repository. That being said, I'd be happy to specify one explicitly after checking with everybody.

Reply:
I can change it for `.format` otherwise.